collections/hrsi-wds.yaml

```diff
@@ -0,0 +1,64 @@
+Name: High-Resolution Snow & Ice Monitoring; Wet / Dry Snow - Snow state classification
+Description: |
+  The High-Resolution Snow & Ice Monitoring service (HR-S&I) is part of the Copernicus Land Monitoring Service (CLMS). The snow aspect of the service provides products 
+  measuring Snow cover (FSC, FSTOC, FSCOG, GFSC), Snow state conditions (WDS, SWS) and persistent snow area (PSA). There are also ice products that measure ice cover 
+  (RLIE) and aggregated river and lake ice extent (ARLIE).
+  
+  More information about the data can be found [here](https://land.copernicus.eu/pan-european/biophysical-parameters/high-resolution-snow-and-ice-monitoring) and
+  the snow products user manual is located [here](https://land.copernicus.eu/user-corner/technical-library/hrsi-snow-pum). 
+  
+  The Wet/Dry Snow (WDS) product differentiates the snow state conditions within the snow mask defined by the FSCTOC information.
+AdditionalInfoExternal:
+    Title: Additional info 
+    Path: hrsi-wds/README.MD 
+Image: hrsi-wds/hrsi-wds.png       
+Resolution: 60m
+GeographicalCoverage: EEA38+UK
+TemporalAvailability: September 2016 - ongoing
+UpdateFrequency:
+BandInformation:
+  Table:
+        Columns:
+          - Name: Name
+            Title: Name
+          - Name: Description
+            Title: Description  
+          - Name: Unit
+            Title: Unit
+          - Name: Sentinel Hub Units
+            Title: Sentinel Hub Units
+          - Name: Data Values
+            Title: Data Values 
+          - Name: No data
+            Title: No data            
+        Rows:
+          - Name: SSC
+            Description: Snow State Classification
+            Unit: Dimensionless
+            Sentinel Hub Units: DN
+            Data Values: Wet snow - 110, Dry snow - 115, Snow free or patchy snow cover - 120, Radar shadow / layover / foreshortening - 200, Cloud or cloud shadow - 205, Water - 210, Forest - 220, Urban area - 230
+            No data: 255
+          - Name: QCSSC
+            Description: Quality layer
+            Unit: Dimensionless
+            Sentinel Hub Units: DN
+            Data Values: High quality - 0, Medium quality, - 1, Low quality - 2, Minimal quality - 3, Masked - 250
+            No data: 255  
+Contact: "[Copernicus Land Monitoring Service Contact](https://land.copernicus.eu/contact)"
+Provider: "[European Environment Agency](https://www.eea.europa.eu/)"
+ManagedBy: "[Sentinel Hub](https://www.sentinel-hub.com/)"
+Tags:
```

Comment:
add the following tags to have common tags across HRSI products
 `snow`, `hrsi`